src/wp-includes/link-template.php

```diff
@@ -1597,25 +1597,39 @@ function get_delete_post_link( $post = 0, $deprecated = '', $force_delete = fals
  * @since 2.3.0
```

Comment:
The new parameter change needs to be documented.
```suggestion
 * @since 2.3.0
 * @since 6.7.0 The $context parameter was added.
```

Reply:
This still needs to be addressed.